src/App.module.css

```diff
@@ -1,6 +1,6 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
-}
\ No newline at end of file
+.mainContainer{
+  background-color: white;
+  width: 1440px;
```

Comment:
**question:** Will setting `max-width` break the animation?

Reply:
No, but there has to be _a width_ set somewhere because otherwise the width defaults to 0 and nothing is being shown. I'm changing it to min-width now.